Gemfile

```diff
@@ -2,6 +2,11 @@ source "https://rubygems.org"
 gemspec
 
 DEFAULT_RAILS_VERSION = '5.0.2'
-gem "mysql2"
+
+if ENV['RAILS_VERSION'] = '4.2.10'
```

Comment:
🙇 Thanks for this! I had it on my list to figure out why the build was failing but had not dug in yet.